appcli/orchestrators.py

```diff
@@ -577,6 +586,74 @@ def __exec_command(self, command: Iterable[str]) -> CompletedProcess:
         return subprocess.run(command, capture_output=True)
 
 
+class NullOrchestrator(Orchestrator):
+    """
+    Orchestrator for standalone applications.
```

Comment:
Clarify.

Orchestrator which has no services to orchestrate. This is useful for appcli applications which consist only of the launcher container containing various additional CLI command groups.